#  Case Study on ANOVA 

### XYZ Company has offices in four different zones. The company wishes to investigate the following :

#### ● The mean sales generated by each zone.

#### ● Total sales generated by all the zones for each month.

#### ● Check whether all the zones generate the same amount of sales.

#### Help the company to carry out their study with the help of data provided.

In [2]:
from google.colab import files
  
  
uploaded = files.upload()

Saving Sales_data_zone_wise.csv to Sales_data_zone_wise.csv


In [8]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
data_set1=pd.read_csv('Sales_data_zone_wise.csv')
data_set1

Month  Zone - A  Zone - B  Zone - C  Zone - D
0    Month - 1   1483525   1748451   1523308   2267260
1    Month - 2   1238428   1707421   2212113   1994341
2    Month - 3   1860771   2091194   1282374   1241600
3    Month - 4   1871571   1759617   2290580   2252681
4    Month - 5   1244922   1606010   1818334   1326062
5    Month - 6   1534390   1573128   1751825   2292044
6    Month - 7   1820196   1992031   1786826   1688055
7    Month - 8   1625696   1665534   2161754   2363315
8    Month - 9   1652644   1873402   1755290   1422059
9   Month - 10   1852450   1913059   1754314   1608387
10  Month - 11   2004480   1740365   1317869   1970069
11  Month - 12   1537539   1875658   1237722   1460165
12  Month - 13   1254939   1588473   1348629   1733383
13  Month - 14   1821799   1844081   1617376   1872259
14  Month - 15   1128185   1804613   1767047   1234311
15  Month - 16   1491534   1542053   1882159   1590913
16  Month - 17   1182110   1552388   2234469   2180416
17  Month - 18   1770324   1699213   1249821   2364132
18  Month - 19   1181635   1959268   2009168   1821882
19  Month - 20   1305972   1559670   2107949   2151008
20  Month - 21   1570152   2013615   1540016   2265814
21  Month - 22   1481619   1527574   2255729   2295079
22  Month - 23   1887694   1992155   1348387   1459683
23  Month - 24   1880820   1752873   2098463   2052591
24  Month - 25   1256333   1622671   1521792   1695122
25  Month - 26   1422853   1715465   1853636   1520406
26  Month - 27   1384426   1983163   1611169   1289160
27  Month - 28   1616640   1547991   2128022   2178267
28  Month - 29   1310654   1660092   1947119   1854412

# Insights:
### The above data shows the data of sales in various zones, Zone A, Zone B, Zone C and Zone D for various months

## 1. Calculate the Mean sales generated by each Zone

In [9]:
data_set1.describe()

Zone - A      Zone - B      Zone - C      Zone - D
count  2.900000e+01  2.900000e+01  2.900000e+01  2.900000e+01
mean   1.540493e+06  1.755560e+06  1.772871e+06  1.842927e+06
std    2.619401e+05  1.683899e+05  3.331937e+05  3.750165e+05
min    1.128185e+06  1.527574e+06  1.237722e+06  1.234311e+06
25%    1.305972e+06  1.606010e+06  1.523308e+06  1.520406e+06
50%    1.534390e+06  1.740365e+06  1.767047e+06  1.854412e+06
75%    1.820196e+06  1.875658e+06  2.098463e+06  2.180416e+06
max    2.004480e+06  2.091194e+06  2.290580e+06  2.364132e+06

## Insights:
### The above table shows the details of the Sales. We need to investigate on the mean sales of each zone. Below are the details:
#### Zone A has a mean sales of 1540493.1379310344	
#### Zone B has a mean sales of 1755559.5862068965	
#### Zone C has a mean sales of 1772871.0344827587	
#### Zone D has a mean sales of 1842926.7586206896
### From the Above, Zone D has the highest mean value and Zone A has the lowest mean value
### Thus, from the primary analysis, the means are not the same.


## 2. Total sales generated by all the zones for each month.

In [10]:
data_set1
data_set1['Total'] = data_set1['Zone - A']+ data_set1['Zone - B']+ data_set1['Zone - C']+data_set1['Zone - D']
data_set1

Month  Zone - A  Zone - B  Zone - C  Zone - D    Total
0    Month - 1   1483525   1748451   1523308   2267260  7022544
1    Month - 2   1238428   1707421   2212113   1994341  7152303
2    Month - 3   1860771   2091194   1282374   1241600  6475939
3    Month - 4   1871571   1759617   2290580   2252681  8174449
4    Month - 5   1244922   1606010   1818334   1326062  5995328
5    Month - 6   1534390   1573128   1751825   2292044  7151387
6    Month - 7   1820196   1992031   1786826   1688055  7287108
7    Month - 8   1625696   1665534   2161754   2363315  7816299
8    Month - 9   1652644   1873402   1755290   1422059  6703395
9   Month - 10   1852450   1913059   1754314   1608387  7128210
10  Month - 11   2004480   1740365   1317869   1970069  7032783
11  Month - 12   1537539   1875658   1237722   1460165  6111084
12  Month - 13   1254939   1588473   1348629   1733383  5925424
13  Month - 14   1821799   1844081   1617376   1872259  7155515
14  Month - 15   1128185   1804613   1767047   1234311  5934156
15  Month - 16   1491534   1542053   1882159   1590913  6506659
16  Month - 17   1182110   1552388   2234469   2180416  7149383
17  Month - 18   1770324   1699213   1249821   2364132  7083490
18  Month - 19   1181635   1959268   2009168   1821882  6971953
19  Month - 20   1305972   1559670   2107949   2151008  7124599
20  Month - 21   1570152   2013615   1540016   2265814  7389597
21  Month - 22   1481619   1527574   2255729   2295079  7560001
22  Month - 23   1887694   1992155   1348387   1459683  6687919
23  Month - 24   1880820   1752873   2098463   2052591  7784747
24  Month - 25   1256333   1622671   1521792   1695122  6095918
25  Month - 26   1422853   1715465   1853636   1520406  6512360
26  Month - 27   1384426   1983163   1611169   1289160  6267918
27  Month - 28   1616640   1547991   2128022   2178267  7470920
28  Month - 29   1310654   1660092   1947119   1854412  6772277

## Insights:
### From the above data, Month -4 has the highest sales (8174449) and Month-13 has the lowest sales(5925424)

## 3. Check whether all the zones generate the same amount of sales.

#### Here, we have one independent variable, Amount of sales, thus we can analyze using One way ANOVA.
#### The below are the desired results:
#### Null Hypothesis:Same amount of sales for all Zones
#### Alternate Hypothesis:Not the same amount of sales for all Zones

In [11]:
import scipy
from scipy.stats import f_oneway
import scipy.stats as stats
fvalue, pvalue = stats.f_oneway(data_set1['Zone - A'],data_set1['Zone - B'],data_set1['Zone - C'],data_set1['Zone - D'])
print('The F value is', fvalue)
print('The p-value is', pvalue)

The F value is 5.672056106843581
The p-value is 0.0011827601694503335


## Insights:
#### The p value obtained from ANOVA Test is 0.0011827601694503335 which is pretty less than 0.05, therefore, we can conclude that the Zones generate different amount of sales. Thus Alternate Hypothesis is applicable here.